spec/support/rails_template_with_data.rb

```diff
@@ -318,8 +318,8 @@
     ActiveAdmin::Comment.create!(
       namespace: :admin,
       author: AdminUser.first,
-      body: "Test comment #{i}",
-      resource: categories.sample,
+      body: "Test comment \#{i}",
```

Comment:
We're also backporting, probably unintentionally, https://github.com/activeadmin/activeadmin/pull/5136 here. I don't have a problem with it but I think backporting the proper commit can make git history a bit less obscure.